framework/src/st_things/things_stack/cloud/cloud_manager.c

```diff
@@ -1795,12 +1795,7 @@ void *cloud_data_cb_esm(es_cloud_prov_data_s *event_data)
 		g_qis_cloud_thread_running = CISESS_BUSY;
 
 		if ((cloned_data = clone_data_add_timeout(event_data, NULL)) != NULL) {
-#ifdef __ST_THINGS_RTOS__
-			int retp = pthread_create_rtos(&cthread_handler, NULL, (pthread_func_type) ci_connection_init_loop, (void *)cloned_data, THINGS_STACK_CICONNETION_INIT_THREAD);
-#else
-
-			int retp = things_thread_create(&cthread_handler, NULL, (pthread_func_type) ci_connection_init_loop, (void *)cloned_data) != 0)
-#endif
+			int retp = pthread_create_rtos(&cthread_handler, NULL, (pthread_func_type) ci_connection_init_loop, (void *)cloned_data, 1024 * 4, "ST_THINGS_STACK_CLOUD_CONNECT");
```

Comment:
Please remove a space between typecasting and variable.
```
(pthread_func_type)ci_connection_init_loop,
```

Reply:
OK, Thanks.